prowler/providers/kubernetes/services/core/core_service.py

```diff
@@ -144,18 +142,16 @@ def _in_worker_node(self):
             )
 
 
-@dataclass
-class Container:
+class Container(BaseModel):
     name: str
     image: str
     command: Optional[List[str]]
     ports: Optional[List[dict]]
     env: Optional[List[dict]]
-    security_context: Optional[V1SecurityContext]
```

Comment:
Why this? Just asking

Reply:
It is necessary to change it to BaseModel.